sdk/storage/Azure.Storage.Blobs.Batch/BreakingChanges.txt

```diff
@@ -3,4 +3,4 @@ Breaking Changes
 
 Next Preview
 --------------------------
-- New Azure.Storage.Blobs.Batching client library.
+- New Azure.Storage.Blobs.Batch client library.
```

Comment:
We should add an entry in this file that we're changing the name since this is definitely a breaking change.  😄